server/index.js

```diff
@@ -15,7 +15,7 @@ config();
 const isProduction = process.env.NODE_ENV === 'production';
 const app = express();
 const log = debug('dev');
-const port = process.env.PORT || 5040;
+const port = process.env.PORT || 5000;
```

Comment:
Is there a reason why this was changed?

Reply:
Yeah, was having too many err:addressinuse